code-style/spotbugs-exclude.xml

```diff
@@ -26,4 +26,8 @@
             <Bug pattern="CT_CONSTRUCTOR_THROW"/>
         </Or>
     </Match>
+    <Match>
+        <Class name="sleeper.statestore.committer.MultiThreadedStateStoreCommitter" />
+        <Bug pattern="DM_GC" />
+    </Match>
```

Comment:
To ignore specific failures like this, we use a SuppressFBWarnings annotation. That's in the Maven dependency com.github.spotbugs:spotbugs-annotations.